index.html

```diff
@@ -1229,32 +1265,26 @@ <h5 class="modal-title-craft"></h5>
                                             </tr>
 
                                             <tr class="dyn_filter_required">
-                                                <td name='dyn_notch_range'>
-                                                    <label>Dyn Notch
-                                                        <br>Range</label>
-                                                    <select>
-                                                        <!-- list generated here -->
-                                                    </select>
-                                                </td>
-                                                <td name="dyn_notch_width_percent">
-                                                    <label>Dyn Notch
-                                                        <br>Width (percent)</label>
-                                                    <input type="number" step="0.1" min="0" max="999.00" />
+                                            <thead>
+                                               <tr>
+                                                <th scope="row" colspan="4">Dynamic notch filters</th>
+                                               </tr>
+                                           </thead>
+                                                <td name='dynNotchCount'>
+                                                    <label>Count/Width</label>
+                                                    <input type="number" step="1" min="0" max="10" />
                                                 </td>
-                                                <td name='dyn_notch_q'>
-                                                    <label>Dyn Notch
-                                                        <br>Q</label>
-                                                    <input type="number" step="0.01" min="0" max="999.00" />
+                                                <td name="dynNotchQ">
+                                                    <label>Q factor</label>
+                                                    <input type="number" step="1" min="0" max="255" />
```

Comment:
Here is max of 255 for Notch Q, thats correct @ctzsnooze ?

Reply:
Right now `dyn_notch_q` can go up to 1000 (min=1, max=1000).

Reply:
@McGiverGim I think it's OK as is, but really haven't a clue.  Please let me know if there's anything that needs changing.

Reply:
@ctzsnooze here is max value of 255 for Q when it should be 999.0 thought.